src/connections/ProjectPitch.ts

```diff
@@ -0,0 +1,132 @@
+import { createSelector } from "reselect";
+
+import { EntityIndexFilterKey } from "@/connections/Entity";
+import {
+  adminProjectPitchesIndex,
+  EntityIndexQueryParams,
+  projectPitchesGetUUIDIndex
+} from "@/generated/v3/entityService/entityServiceComponents";
+import { ProjectPitchDto } from "@/generated/v3/entityService/entityServiceSchemas";
+import {
+  adminProjectPitchesIndexFetchFailed,
+  adminProjectPitchesIndexIndexMeta,
+  projectPitchesGetUUIDIndexFetchFailed,
+  projectPitchesGetUUIDIndexIsFetching
+} from "@/generated/v3/entityService/entityServiceSelectors";
+import { getStableQuery } from "@/generated/v3/utils";
+import ApiSlice, { ApiDataStore, PendingErrorState, StoreResourceMap } from "@/store/apiSlice";
+import { Connection } from "@/types/connection";
+import { connectionLoader } from "@/utils/connectionShortcuts";
+import { selectorCache } from "@/utils/selectorCache";
+
+const projectPitchesSelector = () => (store: ApiDataStore) =>
+  store["projectPitches"] as StoreResourceMap<ProjectPitchDto>;
+
+type ProjectPitchConnection = {
+  isLoading: boolean;
+  requestFailed: PendingErrorState | null;
+  isSuccess: boolean;
+  projectPitch: ProjectPitchDto | any;
```

Comment:
Why is this `any` here? We should be able to avoid uses of `any` in new v3 code.